packages/nimble-components/CONTRIBUTING.md

```diff
@@ -445,10 +445,11 @@ When creating a new component, create a `*-matrix.stories.ts` Storybook file to
 
 Most user-visible strings displayed by Nimble components are provided by the client application and are expected to be localized by the application if necessary. However, some strings are built into Nimble components and are provided only in English. An application can provide localized versions of these strings by using design tokens set on label provider elements.
 
-There are currently 2 label providers:
+There are currently 3 label providers:
 
 -   `nimble-label-provider-core`: Used for labels for all components besides the table
 -   `nimble-label-provider-table`: Used for labels for the table (and table sub-components / column types)
+-   `nimble-label-provider-rich-text-editor`: Used for labels for the rich text editor
```

Comment:
Similar to how the `nimble-label-provider-table` is used for all table related features we should name this provider `nimble-label-provider-rich-text` to support all possible rich-text related features current (`rich-text-editor` and `rich-text-viewer`) and potentially future.

We should propagate that consistently across this PR in all names, comments, etc. ie `nimble-label-provider-rich-text`

Reply:
Updated